spec/models/manageiq/providers/openshift/container_manager/refresher_spec.rb

```diff
@@ -185,18 +197,18 @@ def normal_refresh
       it "archives objects" do
         expect(ContainerProject.count).to eq(8)
         expect(ContainerProject.where(:deleted_on => nil).count).to eq(7)
-        expect(ContainerImage.count).to eq(43) # should be 44
-        expect(ContainerImage.where(:deleted_on => nil).count).to eq(43) # should be 44
+        expect(ContainerImage.count).to eq(38) # should be 39
+        expect(ContainerImage.where(:deleted_on => nil).count).to eq(38) # should be 39
```

Comment:
can you expand the comment to explain why it's 1 less and why we're unsure that's right?
https://github.com/ManageIQ/manageiq-providers-openshift/pull/18#discussion_r121278779